extensions/cloud/portability-cloud-google/src/main/java/org/datatransferproject/cloud/google/GoogleCloudUtils.java

```diff
@@ -0,0 +1,46 @@
+package org.datatransferproject.cloud.google;
+
+import com.google.cloud.Timestamp;
+import com.google.cloud.datastore.Blob;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.Key;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.ObjectOutputStream;
+import java.util.Map;
+
+public class GoogleCloudUtils {
+
+  // Environment variable where GCP project ID is stored. The value is set in
+  // config/k8s/api-deployment.yaml.
+  private static final String GCP_PROJECT_ID_ENV_VAR = "GOOGLE_PROJECT_ID";
+
+  static Entity.Builder createEntityBuilder(Key key, Map<String, Object> data) throws IOException {
```

Comment:
Let's add quick documentation to this, it's useful in many places.

Reply:
Done